p2p/peer_tracker_test.go

```diff
@@ -42,13 +45,17 @@ func TestPeerTracker_GC(t *testing.T) {
 
 	require.Nil(t, p.trackedPeers[pid1])
 	require.Nil(t, p.disconnectedPeers[pid3])
+
+	peers, _ := mockPeerStore.Load(context.Background())
+	require.Equal(t, pid2, p.trackedPeers[pid2].peerID)
+	require.Equal(t, peers[0].ID, p.trackedPeers[pid2].peerID)
```

Comment:
is this deterministic?

Reply:
i'd also assert the expected len

Reply:
It is deterministic cuz we specify the peer scores in line 30/31

Reply:
ok please use assert not require as this is an assertion.